main.go

```diff
@@ -2,6 +2,15 @@ package main
 
 import (
 	"go.k6.io/k6/cmd"
+
+	// Blank-importing golang.org/x/crypto/x509roots/fallback bundles a set of
+	// root fallback certificates from Mozilla into the resulting binary. This
+	// allows the program to run in environments where the system root
+	// certificates are not available, for example inside a minimal container.
+	// These are _fallbacks_, meaning that if the system _does have_ a set of
+	// root certificates, those will be given priority. The binary size will
+	// increase a little (~ 220 kB).
```

Comment:
Would you mind removing the size details? We have no guarantee that this will be always the case and this isn't really important here

```suggestion
	// root certificates, those will be given priority.
```